packages/extension-ui/src/partials/AccountSelection.tsx

```diff
@@ -55,14 +55,16 @@ function AccounSelection({
   }, [areAllAccountsSelected, noAccountSelected]);
 
   const _onSelectAllToggle = useCallback(() => {
+    onChange && onChange(true);
+
```

Comment:
As before - please use plain if's when not assigning the return value to the variable.